implementations/python/provekit-lift-py-pytest-witness/tests/test_witness.py

```
@@ -71,6 +77,159 @@ def test_forged_passed_witness_over_failing_code_is_caught_by_recompute(tmp_path
+        except json.JSONDecodeError:
+            continue
+    raise AssertionError(f"no JSON-RPC reply; stderr={proc.stderr}")
+
+
+def test_resolve_witness_rpc_returns_body_that_addresses_to_cid_from_package(tmp_path):
+    import base64
+    from provekit_pytest_witness import write_witness_package
+    from provekit_lift_py_tests.canonicalizer import blake3_512_of
+    proj = _project(tmp_path, GOOD)
+    w = run_and_witness(proj, "test_add.py", CODE)
+    pkg = tmp_path / "wpkg"; write_witness_package([w], str(pkg))
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)